app/controllers/search_controller.rb

```diff
@@ -1,5 +1,6 @@
 class SearchController < ApplicationController
   def index
-    @results = Item.all
+    search_term = params[:search]
+    @results = helpers.search_for_items(search_term)
```

Comment:
In my opinion if the search bar is empty, the page should display all items.

We should probably ask the PO (@LucasDerReisende ) what the customer wants...

Reply:
We thought about this and decided to show nothing. If you want to see all Items you can visit the landing page.